packages/core/src/sass/app/_common/components/notification-banner.scss

```diff
@@ -0,0 +1,79 @@
+.notification-banner {
+    position: relative;
+    display: flex;
+    background-color: var(--general-main-1);
+    color: var(--text-prominent);
+    border-radius: $BORDER_RADIUS;
+    box-shadow: 0 4px 6px 0 var(--shadow-menu), 0 4px 6px 0 var(--shadow-menu);
+    max-width: 348px;
+    height: 124px;
+    transition: transform 0.25s ease, opacity 0.25s linear;
+    z-index: 10;
+    margin-bottom: 16px;
+
+    &--enter-done {
+        opacity: 1;
+        transform: translateY(0);
+    }
+    &--enter,
+    &--exit {
+        opacity: 0;
+        transform: translateY(-16px);
+    }
+    &--left {
+        padding: 1.6rem 0 1.6rem 1.6rem;
+        width: 210px;
+    }
+    &--right {
+        position: relative;
+        width: 138px;
+    }
+    &__title {
+        font-size: 16px;
```

Comment:
can we use `rem` for font size instead of px?

Reply:
Sure, will do.